python/practices/practice-3/src/constant/constant_file.py

```diff
@@ -0,0 +1,2 @@
+# Constant of file words.txt
+word_file = 'words.txt'
```

Comment:
- The folder name should be `constants` (in plural form).
- The file name is not good, you can use `constants.py`.
- Missing docstring for module.
-  Variable name not good.

Reply:
Can you suggest me any other variable name?

Reply:
Done